src/java.desktop/share/classes/java/awt/ScrollPane.java

```diff
@@ -810,52 +827,3 @@ public AccessibleRole getAccessibleRole() {
     } // class AccessibleAWTScrollPane
 
 }
-
-/*
- * In JDK 1.1.1, the pkg private class java.awt.PeerFixer was moved to
```

Comment:
As I mentioned in the OLD review thread for hg, this definitely needs a CSR
and it needs to be called out.
I think it should be separated out from this fix which is about fixing doclint warnings but here you are making an incompatible change. Let's not mix the two.